llvm-spirv/lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -288,6 +288,49 @@ void SPIRVRegularizeLLVMBase::expandVIDWithSYCLTypeByValComp(Function *F) {
       nullptr, &Attrs, true);
 }
 
+// intel/llvm customization
+void SPIRVRegularizeLLVMBase::finishSROACooperativeMatrix(Module *M) {
+  for (auto &F : *M) {
+    if (!F.getName().starts_with("_Z19__spirv_AccessChain"))
+      continue;
+    for (auto I = F.user_begin(), E = F.user_end(); I != E;) {
+      if (auto *CI = dyn_cast<CallInst>(*I++)) {
+        Instruction *Ptr =
+            dyn_cast<Instruction>(CI->getArgOperand(0)->stripPointerCasts());
+        if (!Ptr || !isa_and_nonnull<AllocaInst>(Ptr))
```

Comment:
```suggestion
        if (!Ptr || !isa<AllocaInst>(Ptr))
```

You have already checked that `Ptr` is not `nullptr` and can use regular `isa`